### PR TITLE
Changes md-text-float to md-input-container

### PR DESCRIPTION
**Pull request by joaojusto**
- Updates angular material to lastest version;
- Removes custom angular mateiral build;
- Adds angular material to bower;
- Changes md-text-float to md-input-container;

### CODE REVIEW

File: app/assets/stylesheets/components/_theme.css.scss [hunk 1]
```diff
@@ -122,6 +122,11 @@ body {
   }
 }
 
+md-card {
+  margin: 8px;
+  padding: 8px;
```
Comment (gabrielpoca) [inline app/assets/stylesheets/components/_theme.css.scss]: why is this necessary? these rules are already there

**Comment by gabrielpoca** [comment]
:+1:

File: app/assets/stylesheets/components/_theme.css.scss (hunk 1 above)
Comment (joaojusto) [inline app/assets/stylesheets/components/_theme.css.scss]: They remove them this version
